samples/BenchmarkDotNet.Samples.FSharp/BenchmarkDotNet.Samples.FSharp.fsproj

```diff
@@ -24,5 +24,6 @@
     <PackageReference Update="FSharp.Core" Version="4.6.0" />
     <PackageReference Update="System.ValueTuple" Version="4.5.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
+    <PackageReference Include="System.Private.Uri" Version="4.3.2" />
```

Comment:
You shouldn't do this.  You don't use it.  What brought it in?  Could you update something higher up the stack to avoid bringing it completely?